src/host/renderData.cpp

```diff
@@ -26,6 +26,16 @@ Microsoft::Console::Types::Viewport RenderData::GetViewport() noexcept
     return gci.GetActiveOutputBuffer().GetViewport();
 }
 
+// Routine Description:
+// - Retrieves the end position of the text buffer. We use
+//   the cursor position as the text buffer end position
+// Return Value:
+// - COORD of the end position of the text buffer
+COORD RenderData::GetTextBufferEndPosition() const noexcept
+{
+    return GetCursorPosition();
+}
+

```

Comment:
Actually, in Terminal::GetTextBufferEndPosition, I actually do not use cursor position anymore. Because that both Terminal and conhost's RenderData implement IBaseData, I need this method definition, but it is not used in conhost actually.